internal/api/approval/approval_test.go

```diff
@@ -135,6 +136,57 @@ func TestContentsUrlToRelDir(t *testing.T) {
 	})
 }
 
+func TestFindCoAuthors(t *testing.T) {
+	tests := map[string]struct {
+		message  string
+		expected []string
+	}{
+		"when no co-authors in commit message": {
+			message:  "feat: awesome new feature",
+			expected: []string{},
+		},
+		"when one co-author in commit message": {
+			message: `feat: awesome new feature
+
+			Co-authored-by: John Doe <12345678+john-doe@users.noreply.github.com>`,
+			expected: []string{"john-doe"},
+		},
+		"when one co-author in commit message (legacy format)": {
+			message: `feat: awesome new feature
+
+			Co-authored-by: John Doe <john-doe@users.noreply.github.com>`,
+			expected: []string{"john-doe"},
+		},
+		"when multiple co-authors in commit message": {
+			message: `feat: awesome new feature
+
+			Co-authored-by: John Doe <12345678+john-doe@users.noreply.github.com>
+			Co-authored-by: Jane Doe <87654321+jane-doe@users.noreply.github.com>`,
+			expected: []string{"john-doe", "jane-doe"},
+		},
+		"when multiple co-authors in commit message (legacy format)": {
+			message: `feat: awesome new feature
+
+			Co-authored-by: John Doe <12345678+john-doe@users.noreply.github.com>
+			Co-authored-by: Jane Doe <jane-doe@users.noreply.github.com>`,
+			expected: []string{"john-doe", "jane-doe"},
+		},
+		"when one of the co-authors uses unsupported format": {
+			message: `feat: awesome new feature
+
+			Co-authored-by: John Doe <john@doe.com>
+			Co-authored-by: Jane Doe <87654321+jane-doe@users.noreply.github.com>`,
+			expected: []string{"jane-doe"},
+		},
+	}
+
+	for name, tt := range tests {
+		t.Run(name, func(t *testing.T) {
+			require.Equal(t, findCoAuthors(tt.message), tt.expected)
```

Comment:
Imho safer to use [ElementsMatch](https://pkg.go.dev/github.com/stretchr/testify/require#ElementsMatch) as it ignores the order of elements.